cmd/api/request/auth.go

```diff
@@ -14,7 +14,7 @@ type Credentials struct {
 // Login validates login request
 func Login(c *gin.Context) (*Credentials, error) {
 	cred := new(Credentials)
-	if err := c.ShouldBindJSON(cred); err != nil {
+	if err := c.BindJSON(cred); err != nil {
```

Comment:
This will write two header to times (BindJSON writes to header, and then apperr.response) resulting in a console log and conflicting error codes.